DESCRIPTION

```diff
@@ -13,4 +13,8 @@ Description: An interface for Open Trip Planner <http://www.opentripplanner.org/
 License: GPL-3
 Encoding: UTF-8
 LazyData: true
+Imports:
+    checkmate,
```

Comment:
Great to see use of a proper data checking framework. Note `stopifnot()` can also go far.